README.md

```diff
@@ -70,19 +70,23 @@ WORKER_DEBUG=mystuff aio asset-compute test-worker
 * [`aio asset-compute devtool`](#aio-asset-compute-devtool)
 * [`aio asset-compute run-worker FILE RENDITION`](#aio-asset-compute-run-worker-file-rendition)
 * [`aio asset-compute test-worker [TESTCASE]`](#aio-asset-compute-test-worker-testcase)
+* [`aio asset-compute tw [TESTCASE]`](#aio-asset-compute-tw-testcase)
```

Comment:
Is `tw` a shortcut for test-worker?

Reply:
This and the rest of the changes to the readme are not changes I made, but slight changes in the way @oclif/core vs @oclif/dev-cli generates the readme from code.
Unfortunately it is also enumerating aliases so in some cases this gets pretty verbose.  I am planning to send a pr to oclif to correct this.

Reply:
https://github.com/adobe/aio-cli-plugin-asset-compute/blob/master/src/commands/asset-compute/test-worker.js#L166